### PR TITLE
Benchmark refactor

### PR DESCRIPTION
**Pull request by hugomg**
* Parameterize benchmark N with argv
* New scripts to run the benchmarks

### CODE REVIEW

File: benchmarks/makefile [hunk 1]
```diff
@@ -0,0 +1,11 @@
+.PHONY: default
+
+default:
```
Comment (gligneul) [inline benchmarks/makefile]: This should be `all`.

Comment (hugomg) [reply benchmarks/makefile]: This isn't actually an "all" target. I renamed it to something that is hopefully clearer.

File: benchmarks/matmul/main.lua [hunk 2]
```diff
@@ -1,33 +1,24 @@
...
-local BIG2 = BIG1
-
 local C
-for _ = 1, 2 do
---for _ = 1, 16 do
---for _ = 1, 128 do
---for _ = 1, 1024 do
-    C = matmul.matmul(BIG1, BIG2)
+for _ = 1, REP do
+    C = matmul.matmul(A, A)
```
[the remark this answers is not in the document]
Comment (hugomg) [reply benchmarks/matmul/main.lua]: We could consider doing that after we are done submitting the paper.

[the remark this answers is not in the document]
Comment (gligneul) [reply benchmarks/matmul/main.lua]: Ok, no problem.

File: README.md [hunk 3]
```diff
@@ -84,14 +84,24 @@ $ busted spec/parser_spec.lua  # Run just one of the test suite files
 ```
 ## Running the benchmarks suite
 
-Running all benchmarks in `benchmarks` directory:
+To run of the benchmarks in the benchmarks directory, tun the `benchmarks/run`
```
Comment (gligneul) [inline README.md]:
```suggestion
To run of the benchmarks in the benchmarks directory, execute the `benchmarks/run`
```

**Pull request merged**